Gemfile

```diff
@@ -5,11 +5,16 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in skippy.gemspec
 gemspec
 
+if RUBY_VERSION.start_with?('3.')
```

Comment:
While I wouldn't expect Ruby 4 to be coming any time soon, I think I'd prefer this to be:

```rb
if RUBY_VERSION.to_i > 2
```